### ISSUE TITLE
cannot search or load or release suddenly but still can insert, error is no replica available 

### ISSUE THREAD

**Issue by kkpssr** [milvus-io/milvus#20071]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.1.4
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

MilvusException: (code=1, message=fail to search on all shard leaders, err=All attempts results:
attempt #1:fail to get shard leaders from QueryCoord: no replica available
attempt #2:fail to get shard leaders from QueryCoord: no replica available
attempt #3:fail to get shard leaders from QueryCoord: no replica available
attempt #4:fail to get shard leaders from QueryCoord: no replica available
attempt #5:fail to get shard leaders from QueryCoord: no replica available
attempt #6:fail to get shard leaders from QueryCoord: no replica available
attempt #7:fail to get shard leaders from QueryCoord: no replica available
attempt #8:context deadline exceeded

### Steps To Reproduce

```markdown
just happened after one insert operation and no idea how to reproduce
```

### Milvus Log

MilvusException: (code=1, message=fail to search on all shard leaders, err=All attempts results:
attempt #1:fail to get shard leaders from QueryCoord: no replica available
attempt #2:fail to get shard leaders from QueryCoord: no replica available
attempt #3:fail to get shard leaders from QueryCoord: no replica available
attempt #4:fail to get shard leaders from QueryCoord: no replica available
attempt #5:fail to get shard leaders from QueryCoord: no replica available
attempt #6:fail to get shard leaders from QueryCoord: no replica available
attempt #7:fail to get shard leaders from QueryCoord: no replica available
attempt #8:context deadline exceeded

**Comment by yanliang567** [comment]
Could you please refer this [script](https://github.com/milvus-io/milvus/tree/master/deployments/export-log) to export the whole Milvus logs for investigation?
/assign @kkpssr

**Comment by kkpssr** [comment]
> Could you please refer this [script](https://github.com/milvus-io/milvus/tree/master/deployments/export-log) to export the whole Milvus logs for investigation? /assign @kkpssr

It is available now, maybe because i search instantly just after milvus service start and some function is not ready, and after 10min , i search  again and succeed.

**Comment by yanliang567** [comment]
okay, i will close tthis issue, and feel free to file a new one if any new issues

**Comment by kkpssr** [comment]
i mean the problem can be dealed but i still can reprouduce just by restart milvus service and i should load index manually, i think maybe is bug? because normally u do not need to load before search.



---Original---
From: ***@***.***&gt;
Date: Wed, Oct 26, 2022 12:03 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [milvus-io/milvus] cannot search or load or release suddenly butstill can insert, error is no replica available  (Issue #20071)




 
okay, i will close tthis issue, and feel free to file a new one if any new issues
